playbooks/templates/.env.j2

```diff
@@ -227,9 +227,17 @@ SKYNET_ACCOUNTS_PORT=3000
 # REQUIRED: manually defined by user, or default will be used
 BLOCKER_LOG_LEVEL={{ webportal_common_config.blocker_log_level }}
 
-# OPTIONAL: manually defined by user, can be blank
-BLOCKER_NGINX_CACHE_PURGE_LIST={{ webportal_common_config.blocker_nginx_cache_purge_list | default('') }}
-BLOCKER_NGINX_CACHE_PURGE_LOCK={{ webportal_common_config.blocker_nginx_cache_purge_lock | default('') }}
+# List of portal urls from which to sync the blocklist periodically
+#
+# OPTIONAL: manually defined by user
+BLOCKER_PORTALS_SYNC={{ webportal_common_config.blocker_portals_sync |
+default('') }}
+
+# OPTIONAL: manually defined by user
```

Comment:
Maybe we should comment those as:
Kept for Ansible playbooks backwards compatibility with older Blocker versions.